### ISSUE TITLE
Concatinated words after stripping html tags

### ISSUE THREAD

**Issue by sandrohoerler** [weblyzard/inscriptis#13]
Example: https://cross-works.net/uber-uns/

Helmut SprollManaging Director
or
Sivan FadelliBuchhaltung & Administration

[cross-html.txt](https://github.com/weblyzard/inscriptis/files/1851973/cross-html.txt)
[inscriptis_output.txt](https://github.com/weblyzard/inscriptis/files/1851976/inscriptis_output.txt)

**Comment by AlbertWeichselbraun** [comment]
This layout problem is caused by an external style file (style.min.css) that sets the `display: block` property for the `person-name` class.

```css
.fusion-person .person-desc .person-author .person-name, .fusion-person .person-desc .person-author .person-title {
    display: block;
}
```

Inscriptis already handles `display: block` correctly, but only if
 1. the css is embedded in the html, and
 2. the style attribute rather than classes are used.

Addressing the 2. issue, doesn't seem to be too much of a problem, but retrieving external css files adds another layer of complexity (especially since many web servers tend to throttle or block access from from non-browser user agents). But if this is a go/no-go criteria for multiple use cases we will implement it as well.

**Comment by AlbertWeichselbraun** [comment]
This is fixed in version 0.4.1.1+.
